src/ScrumBoardItBundle/Resources/views/Print/Poc/template_default.html.twig

```diff
@@ -0,0 +1,10 @@
+{% extends 'ScrumBoardItBundle::Print/Task/template_default.html.twig' %}
+
+{% block content %}
+    <div class="left">
+        <h2>P<br/>O<br/>C<br/></h2>
```

Comment:
<h2> isn't mean to style your text, but specify the weight of it.
<br /> isn't mean to template your design.